root-packages/nexttrace-enhanced/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/OwO-Network/nexttrace-enhanced
+TERMUX_PKG_DESCRIPTION="An open source visual routing tool that pursues light weight"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.1.8-beta.1"
+TERMUX_PKG_SRCURL=https://github.com/OwO-Network/nexttrace-enhanced/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=128303c45e8bf1fcd89efb0a0b0ab8b6745d91c787a902789173e44601ab0af9
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+        termux_setup_golang
+
+        go mod init || :
+        go mod tidy
+        countryCode=$(curl -s "http://ip-api.com/line/?fields=countryCode")
```

Comment:
What is this/what is it suppose to do? 
Since packages are built in docker image in CI the countryCode will be static in any case so I'm not sure what the purpose of curl'ing to get it is

Reply:
https://github.com/OwO-Network/nexttrace-enhanced/blob/5a3161536b23d54ee3d410ad6f3a293146ae35a7/nt_install.sh#L41